src/test/java/com/google/cloud/spanner/pgadapter/AbortedMockServerTest.java

```diff
@@ -237,8 +244,6 @@ public void testQueryHint() throws SQLException {
 
   @Test
   public void testQueryWithParameters() throws SQLException {
-    JdbcMockServerTest.setupJsonbResults(mockSpanner);
-
```

Comment:
Moved this to the test class setup method, so these results are available for all tests in this class.